services/ui-src/src/components/tables/getEntityStatus.test.ts

```diff
@@ -231,6 +263,41 @@ describe("Entity status utilities", () => {
       expect(result).toBe(EntityStatuses.CLOSE);
     });
 
+    it("should not return CLOSE for closed initiatives in the SAR", () => {
+      const entity = {
+        id: "entity1",
+        type: "initiative",
+        isInitiativeClosed: true,
+      } as EntityShape;
+
+      const report = {
+        reportType: "SAR",
+        formTemplate: {
+          routes: [
+            {},
+            {},
+            {
+              pageType: PageTypes.DYNAMIC_MODAL_OVERLAY,
+              entityType: "initiative",
+              entityInfo: ["initiative_name", "initiative_wpTopic"],
+              verbiage: {
+                intro: {
+                  section: "",
+                },
+              },
+              initiatives: [entity],
+            },
+          ],
+        },
+      } as ReportShape;
+      const isPdf = false;
+      const ignore: string[] = [];
+
+      const result = getInitiativeStatus(report, entity, isPdf, ignore);
+
+      expect(result).not.toBe(EntityStatuses.CLOSE);
```

Comment:
do we want to check for the expected status here? not closed is good though if not

Reply:
We _could_, but that status kind of depends on whether the reporting info is complete or not ([list of statuses here](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/blob/main/services/ui-src/src/components/tables/EntityStatusIcon.tsx#L10-L15)), and I thought the more important thing to test is that we should _not_ be seeing the closed icon in the SAR. That icon, according to design, should only appear in the WP.